docs/process.md

```diff
@@ -53,7 +53,7 @@ The [issue labels](issues.md#issue-labels) may help you in choosing which issue
    git pull
    ```
 
-1. Create a new branch to push your commits into. If you have push access, name it `{IssueNumber}-{some-keywords}`,
+1. Create a new branch to push your commits into. If you are pushing to the committer repo, name it `{IssueNumber}-{some-keywords}`,
```

Comment:
`have commit access and need to push into the main repo for some reason`

Perhaps the term "committer repo" should be retired, unless there's another "committer repo" I'm unaware of.